src/common/marlin_server.cpp

```diff
@@ -370,6 +370,14 @@ void marlin_server_do_babystep_Z(float offs) {
     babystep.task();
 }
 
+extern void marlin_server_move_axis(float pos, size_t axis) {
+    xyze_float_t position = current_position;
+    position[axis] = pos;
+    current_position[axis] = pos;
+    static std::array<int, 4> feedrates = { MANUAL_FEEDRATE };
```

Comment:
Can we avoid the array, or use a constexpr?

Reply:
Yes, the feedrate will be added as argument to the function